polysemy-plugin/src/Polysemy/Plugin.hs

```diff
@@ -64,6 +64,7 @@ module Polysemy.Plugin
 import Polysemy.Plugin.Fundep
 #if __GLASGOW_HASKELL__ >= 810
 import Polysemy.Plugin.Phases
+import Data.Bool (bool)
```

Comment:
@TheMatten could you take a look at this? I took a guess as to which `bool` is supposed to be used here, but I don't really know what the code does, and `blame` says you wrote the part with `bool`

Reply:
Yeah, it should be that one :+1: